scripts/ci/docker-compose-interactive.yml

```diff
@@ -14,6 +13,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
 
-docker-compose -f scripts/ci/docker-compose.yml run --rm airflow-testing "${@-bash}"
+version: "2.2"
+services:
+  airflow-testing:
+    entrypoint: [ "/bin/bash", "-c" ]
```

Comment:
Is this really worth a file? Also `-c` doesn't sound like it's interactive - why do  I get a bash shell with this?

Reply:
Espeically given this doesn't actually change anything - the base docker-compose file already has this setting

Reply:
Very good point. It's an artifact from earlier tries with tini/dumb-init. When I played with it I had dumb-init set in the main file, and it messed with closing stdin/stdout for interactive runs. But I decided it's not really needed for running tests via docker compose, so I remove this file now.